readme/BREAKING_CHANGES.markdown

```diff
@@ -271,4 +271,33 @@ The first recommendation is to use the updated CSS classes from Clay `.container
 
 This change was made to remove deprecated legacy code from Portal and improve the code consistency and performance
 
+---------------------------------------
```

Comment:
In rebasing this got a conflict; surprised Brian merged the update right above this one because it is not hard-wrapped to 80 columns.